h/models/group.py

```diff
@@ -47,6 +57,17 @@ class GroupMembership(Base):
         nullable=False,
         index=True,
     )
+    roles = sa.Column(
+        JSONB,
+        sa.CheckConstraint(
+            " OR ".join(
+                f"""(roles = '["{role}"]'::jsonb)""" for role in GroupMembershipRoles
```

Comment:
I think using a loop here is not the best choice, it gives the impression that things will just work but they won't.

Listing them all will make it more obvious that a migration is needed as the model definition of the constraint needed a change.

Reply:
Hmm. Yeah, ok, I think I agree with this: changed it to a hard-coded list